--- /tools/Xilinx/Vivado/2024.2/lib/scripts/rdi/features/base/base.tcl.orig	2024-11-20 23:52:22.090667518 +0000
+++ /tools/Xilinx/Vivado/2024.2/lib/scripts/rdi/features/base/base.tcl	2024-11-20 23:53:20.169954528 +0000
@@ -38,6 +38,6 @@
 rdi::disable_console
 
 # work around x11 XSupportsLocale bug
-rdi::x11_workaround
+#rdi::x11_workaround
 
 
--- /tools/Xilinx/Vivado/2024.2/data/ip/xilinx/vitis_net_p4_v2_2/src/sw/drivers/cam_top/src/cam_top.c.orig	2024-11-20 23:55:48.636028610 +0000
+++ /tools/Xilinx/Vivado/2024.2/data/ip/xilinx/vitis_net_p4_v2_2/src/sw/drivers/cam_top/src/cam_top.c	2024-11-20 23:56:56.107159834 +0000
@@ -1195,6 +1195,7 @@
      */
     while (1)
     {
+        Position = 0;
         CamResult = tcam_get_by_response(CtxPtr->PrivateCtxPtr->CamUnion.TcamPtr, ResponsePtr, ResponseMaskPtr, &Position, KeyBufPtr, MaskBufPtr);
 
         if (CamResult == CAM_ERROR_KEY_NOT_FOUND)
*** /tools/Xilinx/Vivado/2024.2/scripts/Vivado_init.tcl.orig	1969-12-31 19:00:00.000000000 -0500
--- /tools/Xilinx/Vivado/2024.2/scripts/Vivado_init.tcl	2025-05-08 13:55:18.223387992 -0400
***************
*** 0 ****
--- 1,2 ----
+ # Enable U280 as 'beta' device (required as of Vivado 2024.2)
+ enable_beta_device xcu280*
