--- /tools/Xilinx/Vivado/2023.2/lib/scripts/rdi/features/base/base.tcl.orig	2023-11-03 16:01:18.290697388 +0000
+++ /tools/Xilinx/Vivado/2023.2/lib/scripts/rdi/features/base/base.tcl	2023-11-03 16:01:29.638691029 +0000
@@ -38,6 +38,6 @@
 rdi::disable_console
 
 # work around x11 XSupportsLocale bug
-rdi::x11_workaround
+#rdi::x11_workaround
 
 
--- /tools/Xilinx/Vivado/2023.2/data/ip/xilinx/vitis_net_p4_v2_0/src/sw/drivers/cam_top/src/cam_top.c.orig	2024-11-17 13:27:40.613408016 +0000
+++ /tools/Xilinx/Vivado/2023.2/data/ip/xilinx/vitis_net_p4_v2_0/src/sw/drivers/cam_top/src/cam_top.c	2024-11-17 13:30:01.323600486 +0000
@@ -1134,6 +1134,7 @@
      */
     while (1)
     {
+        Position = 0;
         CamResult = tcam_get_by_response(CtxPtr->PrivateCtxPtr->CamUnion.TcamPtr, ResponsePtr, ResponseMaskPtr, &Position, KeyBufPtr, MaskBufPtr);
 
         if (CamResult == CAM_ERROR_KEY_NOT_FOUND)
